eventual-cassandra/src/test/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/EventualCassandraTest.scala

```diff
@@ -444,7 +446,7 @@ object EventualCassandraTest {
   }
 
   def statementsOf(segmentNrsOf: SegmentNrsOf[StateT], segments: Segments): EventualCassandra.Statements[StateT] = {
-    implicit val concurrentStateT = ConcurrentOf.fromMonad[StateT]
+    val concurrentStateT: Concurrent[StateT] = ConcurrentOf.fromMonad[StateT]
```

Comment:
Why it was changed from been `implicit`?

Reply:
when type gets exposed (as newer Scala requires), the evaluation of value gets in loop as closest instance it can find is itself, IIUC